staging/src/k8s.io/kubectl/pkg/cmd/get/customcolumn.go

```diff
@@ -241,7 +242,11 @@ func (s *CustomColumnsPrinter) printOneObject(obj runtime.Object, parsers []*jso
 			values, err = parser.FindResults(reflect.ValueOf(obj).Elem().Interface())
 		}
 
-		if err != nil {
+		if err != nil && strings.Contains(fmt.Sprint(err), "array index out of bounds") {
+			// return none when out of bound error like .status.conditions[99].status
+			klog.V(4).InfoS("Parse failed", "err", err)
+			err = nil
```

Comment:
This is wrong approach, we're loosing information that a user name a mistake, he should be informed about it, but we should continue working on it. So I'd suggest printing error but continue with parsing.

Reply:
Updated.
Print the error and continue with parsing.